Python/DSC_event_scraper/event_scraper.py

```diff
@@ -0,0 +1,26 @@
+import requests
+import csv
+import sys
+import json
```

Comment:
I don't think that the imports are properly stacked up. Can you try running your Script through PyLint and review what changes you can do here?